### ISSUE TITLE
Using with pytorch, gives tensorflow error

### ISSUE THREAD

**Issue by sensharma** [comet-ml/issue-tracking#366]
I am trying to use comet_ml (tried versions 3.1.12 and 3.1.11) with pytorch(1.5.0) (python 3.8) on a gpu (cuda 10.1, cudnn 7.6). I am getting warnings such as:
`
COMET WARNING: Failed to convert tensorflow tensor tensor(3.6374, device='cuda:0') to numpy object
COMET WARNING: Failed to convert tensorflow tensor tensor(55.6177, device='cuda:0', grad_fn=<AddBackward0>) to numpy object
`
I am not using tensorflow at all, not even importing it in the module. Is this a bug or does comet_ml have some inherent tensorflow dependency?

The code seems to be running, but does the above affect performance or tracking accuracy in any way?

**Comment by dsblank** [comment]
Interesting. PyTorch must also use a `tensor.numpy()` method to allow conversion like TensorFlow does. So, no, there is no tf dependency, and just a little overhead as comet_ml tries to convert it and fails. If this comes from a logging command, you could first convert it to something generic (like a list of numbers). But this should be fine. We've changed the log level (for next release) so you won't see this warning in the future unless running with `COMET_LOGGING_LEVEL` set to "debug". Thanks for reporting!
